website/content/docs/nia/architecture.mdx

```diff
@@ -46,9 +46,10 @@ discovered IP addresses for a set of Consul services.
 ## Drivers
 
 A driver encapsulates the resources required to communicate the updates to the
-network infrastructure. Terraform is the initial supported driver. It comes
-with a set of providers that [enables supporting](/docs/nia/terraform-modules) a wide variety of
-infrastructure applications.
+network infrastructure. Currently, the Terraform driver and Terraform Cloud
+driver<EnterpriseAlert inline />are the supported [drivers](/docs/nia/network-drivers#terraform).
```

Comment:
> "driver<EnterpriseAlert inline />are"

Looks like there might be a missing space here, though you do the same thing below... does the "<EnterpriseAlert inline />" include a space or something?

If it is missing a space, then the other case needs one as well.

Reply:
Thanks for pointing that out! I removed the space between "drive" and "are" because the Enterprise alert has some padding around it and the space made the gap between the alert and "are" look a bit large to me. Not having the space looked most natural to me.

🤔 it might be visually negligible and less confusing to have the space though. Let me know what your thoughts! Screenshots below for reference

Screenshot of `driver<EnterpriseAlert inline />are` (no space):
<img width="314" alt="Screen Shot 2021-07-20 at 1 58 38 PM" src="https://user-images.githubusercontent.com/6819378/126373063-8717845b-65ee-48a6-8949-3af97592da2b.png">

Screenshot of `driver<EnterpriseAlert inline /> are` (with space):
<img width="307" alt="Screen Shot 2021-07-20 at 2 00 10 PM" src="https://user-images.githubusercontent.com/6819378/126373036-b86eb17c-3315-4f82-9462-e45a29afd59e.png">

Reply:
Yikes. That enterprise alert is way bigger than I expected. 

I don't really think it matters with that giant ENTERPRISE in there... no space needed.

BTW, what's the trick to getting the URL to preview the docs? Been to long.

Reply:
👍 ok I'll keep the no-space for now. If we find it confusing down the line, I can add it!

For getting the URL to preview the docs:
 - For this particular PR, I linked it in the PR description but wasn't explicit enough. Let me fix that
 - In general, I click on the green-check/red-X icon to view the jobs that ran:<img width="428" alt="Screen Shot 2021-07-20 at 2 47 31 PM" src="https://user-images.githubusercontent.com/6819378/126379271-8ed33c1e-aad4-45ae-a6a1-65d09fdce5f6.png">
 And click the "Vercel - consul" job which opens a new page:<img width="445" alt="Screen Shot 2021-07-20 at 2 47 45 PM" src="https://user-images.githubusercontent.com/6819378/126379342-aa4844fc-3b9f-412e-9f84-9b2a1784953a.png">
  And the click the "Visit" button, which will bring you to the preview